mappings/net/minecraft/util/registry/BuiltinRegistries.mapping

```diff
@@ -56,3 +64,7 @@ CLASS net/minecraft/class_5458 net/minecraft/util/registry/BuiltinRegistries
 		ARG 0 registry
 		ARG 1 id
 		ARG 2 value
+	CLASS class_7488 Initializer
```

Comment:
A separate pull request can rename the static methods implementing this functional interface, since they are currently very inconsistent.

Reply:
`initAndGetDefault` can stay, I believe. The only reason I didn't name this `InitAndGetDefaulter` or something is... it looks kinda ugly

Reply:
The functional interface allows the return value to be documented without duplicating the documentation across every method or keeping `andGetDefault` in the method name.

Reply:
Well, in that case we can unify that.